modules/monitoring-moving-monitoring-components-to-different-nodes.adoc

```diff
@@ -75,10 +75,10 @@ data:
         foo: bar*
     openshiftStateMetrics:
       nodeSelector:
-        node-role.kubernetes.io/infra: ""
+        foo: bar
     thanosQuerier:
       nodeSelector:
-        node-role.kubernetes.io/infra: ""
+        foo: bar
```

Comment:
I think we need separate PRs for 4.4 and 4.5
in 4.4, we don't support configure thanosQuerier, so 4.4 and 4.5 are different bug
4.5 need to change nodeSelector to "foo: bar" for openshiftStateMetrics/thanosQuerier
but for 4.4, we only need to change that for openshiftStateMetrics